lib/tasks/generate.rake

```diff
@@ -6,4 +6,12 @@ namespace :generate do
 
     Task::OverdueUserNotificationList.new(year: year, month: period_month).generate
   end
+
+  desc 'Generates CSV of users and their anticipated tasks for a given window. Defaults to the next window'
+  task anticipated_user_tasks: [:environment] do |_task, args|
+    year         = args[:year]&.to_i         || Time.zone.today.year
+    period_month = args[:period_month]&.to_i || Time.zone.today.month
```

Comment:
If this gets run before the start of the window, won't this default to the current rather than the *next* window ?

Reply:
Possibly a bit confusing, but the month/year passed in is the reporting period, not the task's month e.g. Getting the CSV for January's MI would mean passing in 2019 and 1. So the default assumes you want the CSV for the current month's data because you're generating it before the end of the month and ahead of the 1st of next month when the tasks will get generated.